101-vsts-cloudloadtest-rig/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-  "itemDisplayName": "VSTS-CLT-LoadAgent-VMRig",
+  "itemDisplayName": "VSTS-CloudLoadTest-Rig",
```

Comment:
Name shold be more descriptive as this is displayed in Azure.com. You have 60 characters for this field.